src/plots/cartesian/axes.js

```diff
@@ -1137,7 +1133,7 @@ axes.tickText = function(ax, x, hover) {
 };
 
 function tickTextObj(ax, x, text) {
-    var tf = ax.tickfont || ax._gd._fullLayout.font;
+    var tf = ax.tickfont || {};
```

Comment:
Oh. I need to double-check this. 

Removing the `|| {}` should do the trick - as axis `tickfont` already default to layout `font`, but removing `|| {}` makes the [`fonts.json`](https://github.com/plotly/plotly.js/blob/master/test/image/mocks/fonts.json) mock fail. Not sure why.